apps/portal-api/src/modules/services/integration-feeds/integration-feeds.graphql

```diff
@@ -75,12 +75,12 @@ type Connector implements Node & DocumentBase & IntegrationFeed {
   integration_type: IntegrationFeedType!
   product_version: String!
   container_image: String
-  verified: String!
+  verified: Boolean!
   source_code: String
   subscription_link: String
   integration_subtype: ConnectorType!
-  manager_supported: String!
-  playbook_supported: String!
+  manager_supported: Boolean!
```

Comment:
Since this property is now a boolean, you can change the condition using `manager_supported` in `shareable-resource-connector-slug.tsx` file 😁